references/classification/train.py

```diff
@@ -16,9 +16,9 @@
 
 
 try:
-    from torchvision.prototype import models as PM
```

Comment:
I can foresee that in the future if we are going to import prototype datasets as well, we'll introduce another annotation named `PD`... so my question is, why not import the full prototype here? 
I assume the logic behind the way you raise `ImportError` is, if we're having import error with one of them, then we are setting all of them to None.
So I'm thinking about simplify the import to
```
try:
      import torchvision.prototype as prototype
except ImportError:
      prototype = None
```

Reply:
I was going for the minimum number of modifications on this PR. I don't mind refactoring if you think that's needed.